src/app/components/layout/certificate-base-login/certificate-base-login.component.ts

```diff
@@ -69,14 +69,18 @@ export class CertificateBaseLoginComponent implements OnInit {
 
   LoginWithCertificate() {
     if (this.certificateContent.length <= 0
-      && this.form.value.certificateText.length <= 0) {
+      && this.form.get('certificateText').value.length <= 0) {
```

Comment:
May be better to hold value in `const certificateTextValue =  this.form.get('certificateText').value`